packages/liquidator/index.js

```diff
@@ -105,10 +111,8 @@ async function run({
       );
 
     // Setup contract instances. This uses the contract version pulled in from previous step.
-    const financialContract = new web3.eth.Contract(
-      getAbi(liquidatorConfig.contractType, liquidatorConfig.contractVersion),
-      financialContractAddress
-    );
+    const { getAbi } = require(getContractsNodePackageAliasForVerion(liquidatorConfig.contractVersion));
```

Comment:
You might want to assign this to a different name and import the normal getAbi from `@uma/contracts-node` the top. The reason I suggest this is that this causes us to unintentionally use the _versioned_ `getAbi` for all the contracts that come up later in this function. Since the old code used the _current_ abis for these other contracts, it might make sense to keep that consistent.

Does that make sense?

I did something similar in the disputer package.

Reply:
Hhhmm. did you do that in the disputer? my implementation looks identical to what you put in the disputer package. the disputer has:
```ts
// Setup contract instances.
    const { getAbi } = require(getContractsNodePackageAliasForVerion(disputerConfig.contractVersion));
    const financialContract = new web3.eth.Contract(getAbi(disputerConfig.contractType), financialContractAddress);
```

Reply:
Ha, odd, I must've not committed the change. I will change it in a follow-up :).

Reply:
yup, I agree with you to avoid any confusion or unintentionally calling it. what shall we call this? `getVersionedAbi` or what do you think?

Reply:
after discussing this offline we will leave this as it is (for now) and then update both this package and the disputer in the same PR later on.